backend/flindt/feedback/views.py

```diff
@@ -40,8 +40,17 @@ def get_queryset(self):
         """
         Set the queryset here so it filters on user.
         """
-        return super(FeedbackViewSet, self).get_queryset().filter(Q(sender=self.request.user) |
-                                                                  Q(recipient=self.request.user))
+        data = self.request.data
+        feedback_type = data.get('feedbackType', None)
+        if not feedback_type:
+            return None
```

Comment:
raise BadRequest?

btw, this if check makes the function not backwards compatible with the current frontend, so it needs to be coordinated again